core/rollup_l1_cost.go

```diff
@@ -20,17 +20,15 @@ import (
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/params"
 )
 
-var big10 = big.NewInt(10)
-
 var (
 	L1BaseFeeSlot = common.BigToHash(big.NewInt(1))
-	OverheadSlot  = common.BigToHash(big.NewInt(3))
-	ScalarSlot    = common.BigToHash(big.NewInt(4))
-	DecimalsSlot  = common.BigToHash(big.NewInt(5))
+	OverheadSlot  = common.BigToHash(big.NewInt(5))
```

Comment:
Need to confirm that these are the correct storage slots

Reply:
I've done the check, these look correct.

Reply:
Tests do cover these slots indirectly (we have action tests for reading/changing them), and I confirmed the values by debug-stepping through chainops genesis creation to find the storage layout. Not sure where else to find the information.

Reply:
`forge inspect SystemConfig storageLayout`